DeadCrumbs/app/src/main/AndroidManifest.xml

```diff
@@ -56,6 +56,9 @@
         <service
             android:name=".ui.BluetoothService"
             android:theme="@android:style/Theme.Translucent.NoTitleBar" />
+        <service
+            android:name=".ui.DeadReckoningService"
+            android:theme="@android:style/Theme.Translucent.NoTitleBar"/>
```

Comment:
Ret sikker på den her line er unødvendig

Reply:
It compiles fine without line 61. Authors may also delete line 58 as it is the same issue there.